tests/test_image_transforms.py

```diff
@@ -101,6 +101,33 @@ def test_to_pil_image_from_float(self, name, image_shape, dtype):
         with self.assertRaises(ValueError):
             to_pil_image(image)
 
+        # Make sure binary mask remains a binary mask
+        image = np.random.randint(0, 2, image_shape).astype(np.uint8)
+        pil_image = to_pil_image(image)
+        self.assertIsInstance(pil_image, PIL.Image.Image)
+        self.assertEqual(pil_image.size, (5, 4))
```

Comment:
I feel here we miss the objective of the check: `binary mask remains a binary mask` ..?

Reply:
it also looks this check is implemented in `test_to_pil_image_from_mask` below, I am not very sure the reason of this new block here.

Reply:
You're right. I originally put the check here but then moved below as this method is `test_to_pil_image_from_float` so didn't want to use int inputs. Forgot to delete. I'll remove 👍